### ISSUE TITLE
Disconnecting inside a predator has been teleporting people into space

### ISSUE THREAD

**Issue by Michab02** [VOREStation/VOREStation#610]
#### Brief description of the issue
More or less exactly what the title says.
#### What you expected to happen
The prey would be in the pred.
#### What actually happened
The prey was in space.
#### Steps to reproduce
1. Pick up a micro.
2. Attempt to eat them.
3. SPAAAAAAAAAAACE.

Work-around: Grab swallow people like you normally would.

**Comment by SpadesNeil** [comment]
Server revision is missing.

**Comment by Eearslya** [comment]
It *just* happened on the server, so..
```
Server revision: release - 2016-09-30
e4c5b15c3cc768cecda281cafd0f0e9dcaa669c2 
```

**Comment by SpadesNeil** [comment]
Vore in general is spacing people. I suspect the cause may be a recent change to the vore code.

**Comment by SpadesNeil** [comment]
Rix reports this only occurs when you try to eat a micro holder.

Updating steps to reproduce.

**Comment by SpadesNeil** [comment]
https://github.com/VOREStation/VOREStation/pull/613 didn't fix the issue.

Steps to reproduce bug:
1. Pick up micro in hand.
2. Click self.
3. Eat micro.
4. They cannot use 'say' or whatever.

Oddly they are otherwise just fine and not taking damage in this case.

**Comment by Eearslya** [comment]
I know vore code has some special overrides so the prey can continue breathing even if their predator is in space. That's probably why they don't take damage, and it's likely that some oversight on that code is what prevents them from using `say` as well.

Since this bug is only affecting held vore, I searched up for `/obj/item/weapon/holder` (the item used for held mobs) and there was a change to its code not long ago. I've reverted that change for now, and we'll see where that gets us. Compiling and testing now.
